packages/react/src/themes/teams/components/Loader/loaderStyles.ts

```diff
@@ -0,0 +1,85 @@
+import { LoaderProps } from '../../../../components/Loader/Loader'
+import { ComponentStyleFunctionParam, ICSSInJSStyle } from '../../../types'
+import { LoaderVariables } from './loaderVariables'
+
+export default {
+  indicator: ({
+    props: p,
+    variables: v,
+  }: ComponentStyleFunctionParam<LoaderProps, LoaderVariables>): ICSSInJSStyle => ({
+    // Reset existing styles from base theme
+    animationName: 'none',
```

Comment:
I wish we didn't have to reset here all existing styles from the base theme, but I don't have any other proposal. The styles for the base theme seems reasonable for it's usage

Reply:
Yep, it's absolutely not funny, but I don't have a better idea now...